packages/website/src/pages/FileAction.tsx

```diff
@@ -26,6 +26,13 @@ function FileAction() {
   // Support we have a folder, containing a shortcut to a README document,
   // the rInner is README and the rOuter is the folder.
   const { inMost: rInner, outMost: rOuter } = useRender();
+  let [revisionsEnabled, setRevisionsEnabled] = useState(false);
+  function toggleRevisions() {
+    revisionsEnabled = !revisionsEnabled;
+    setRevisionsEnabled(revisionsEnabled);
+  }
+  const revs: Array<gapi.client.drive.Revision> = []
+  const [revisions, setRevisions] = useState(revs);
```

Comment:
```suggestion
  const [revisions, setRevisions] = useStateArray<gapi.client.drive.Revision>([]);
```

Reply:
I don't see `useStateArray` anywhere.

Reply:
My fault! It should be `useState<Array<gapi.client.drive.Revision>>([])`